graphene_django_cud/converter.py

```diff
@@ -155,7 +155,6 @@ def convert_django_field_to_input(
 @convert_django_field_to_input.register(models.SlugField)
 @convert_django_field_to_input.register(models.URLField)
 @convert_django_field_to_input.register(models.GenericIPAddressField)
-@convert_django_field_to_input.register(models.FileField)
```

Comment:
Can you explain this?

Reply:
It was just a tiny cleanup, `FileField` is already handled by `convert_file_field_to_upload` (overwrites this in line 457)